Login.spec.js

```diff
@@ -0,0 +1,105 @@
+describe('Login Page Tests', () => {
+
+  before(() => {
+    // Visit the Login page before running the tests
+    cy.visit('/login');
+  });
+
+  beforeEach(() => {
+    // Clear local storage before each test
+    window.localStorage.clear();
+  });
+
+  it('Should render the login page correctly', () => {
+    // Verify if the logo is displayed
+    cy.get('img[alt="ATS Logo"]').should('be.visible');
+
+    // Check the main headings
+    cy.contains('ATS').should('be.visible');
+    cy.contains('Login to Attendance Tracker System (ATS)').should('be.visible');
+  });
+
+  context('Form Submission Tests', () => {
+
```

Comment:
Consider creating a helper function for form submission. This function can take email and password as parameters and perform the form submission. This will reduce code repetition.